Cargo.toml

```diff
@@ -18,7 +18,8 @@ path = "src/lib.rs"
 serde_json = "1.0"
 serde_repr = "0.1"
 serde = { version = "1.0.101", features = ["derive"] }
-flux = { git = "https://github.com/influxdata/flux", rev = "8e956e67" }
+flux = { git = "https://github.com/influxdata/flux", rev = "1c7cc9f0" }
+libstd = { git = "https://github.com/influxdata/flux", rev = "1c7cc9f0" }
```

Comment:
isn't this just reimporting flux as another name?

Reply:
nevermind just found https://github.com/influxdata/flux/blob/8fd3eae64365b839ed1c1477842cf56b745470fb/libflux/src/libstd/Cargo.toml :)

Reply:
no, there are two libraries in that repo and we have to explicitly get both.